src/config/dynamic-theme-fixes.config

```diff
@@ -23659,6 +23659,25 @@ img.section-bg-absolute
 
 ================================
 
+smallcultfollowing.com
```

Comment:
I have nicer and smaller solution which won't invert whole page and picture.

You can use it here for your PR:
```
smallcultfollowing.com

CSS
*:not(a) {
    color: var(--darkreader-neutral-text) !important;
}

================================


```

Reply:
Great solution, I will update the PR. Thank you!